docs/api/LightroomPartnerAPIsSpec.json

```diff
@@ -540,6 +540,116 @@
         }
       }
     },
+    "/v2/catalogs/{catalog_id}/assets/{asset_id}/renditions": {
+      "post": {
+        "tags": [
+          "assets"
+        ],
+        "summary": "Generate renditions for an original file",
+        "description": "Generate renditions for an original file asynchronously.",
+        "operationId": "generateRenditions",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "X-API-Key",
+            "in": "header",
+            "description": "Client ID (API Key) which is subscribed to the Lightroom APIs through console.adobe.io",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "X-Generate-Renditions",
+            "in": "path",
+            "description": "One or multiple of the supported rendition types separated by ','. Supported rendition types are : ['fullsize', '2560'].",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Authorization",
+            "in": "header",
+            "description": "Bearer [token] - User access token of an authenticated Lightroom customer",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Content-Length",
+            "in": "header",
+            "description": "Length in bytes of the content.",
+            "type": "integer"
+          },
+          {
+            "name": "catalog_id",
+            "in": "path",
+            "description": "Identifier of the catalog in which the asset and revision was created.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "asset_id",
+            "in": "path",
+            "description": "Identifier of the asset for which rendition gets generated.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202 (OK)": {
```

Comment:
the success response will also have a `links` section, right? that needs to be added. we should also add that client needs to poll on that link and accessing it might not immediately return a success response.